### ISSUE TITLE
Preload main feed data on startup

### ISSUE THREAD

**Issue by dtonon** [mikedilger/gossip#750]
When the user unlocks the private key or just skip the login, he faces an empty page with a (useless) "load more" button for some seconds:
![image](https://github.com/mikedilger/gossip/assets/89577423/b3642ab6-624d-4442-91a5-ca0214d6c73b)

Can we start fetching the data immediately when Gossip starts, at least if the user didn't select "Start in offline mode" in a previous session?

If that is not possibile, we can fallback with a [better loading view](https://github.com/mikedilger/gossip/issues/749)

**Comment by dtonon** [comment]
@mikedilger this needs your approval :)

/cc @bu5hm4nn

**Comment by mikedilger** [comment]
I don't think we should load the data immediately, I think the 'loading' screen Bushmann just did is sufficient.

**Comment by bu5hm4nn** [comment]
Well we could load a local cache version of the feed and then update to the live later?

**Comment by dtonon** [comment]
@mikedilger 

> I don't think we should load the data immediately,

Is it for a specific reason? Just to understand your pov.

> Well we could load a local cache version of the feed and then update to the live later?

Actually, if I access "offline" the feed is immediately available, so the db data is already used in this case. Showing it could be a solution, but you will have a refresh in some seconds and since the feed auto scrolls, you will loose the context. I would avoid that.

It's not a critical point of course, just a nice touch.

**Comment by mikedilger** [comment]
I can't reproduce. If I start in offline mode I see all the local notes just fine.  I don't see any useless empty feed.
